charts/customer-center/values.yaml

```diff
@@ -0,0 +1,230 @@
+      from: customer-center@example.com
+      # backend.settings.stmp.to -- Default destination email address
+      to: administration@example.com
+      # backend.settings.stmp.secure -- If true the connection will use TLS when connetion to server
+      secure: true
+      # backend.settings.stmp.requireTLS -- If this is true and secure is false then Nodemailer tries to use STARTTLS even if the server does not advertise support for it. If the connection can not be encrypted then message is not sent
+      requireTLS: true
+    auth:
+      # backend.settings.auth.secret -- Authentication settings
+      secret: my-super-secret
+      # backend.settings.auth.roleAdmin -- Admin role to use
+      roleAdmin: ldap-role-admin
+      # backend.settings.auth.roleEmployee -- Employee role to use
+      roleEmployee: ldap-role-employee
+      # backend.settings.auth.expirePassword -- Timeout in seconds before Redis key expiration
+      expirePassword: 3600
+      # backend.settings.auth.usersSearchBase -- Search base to find users
+      usersSearchBase: ou=staff,cn=accounts,dc=sso,dc=example,dc=com
+      # backend.settings.auth.usersSearchFilter -- Search filter to get users username
+      usersSearchFilter: (uid={{username}})
+      # backend.settings.auth.usersGroupSearchBase -- Search base to find users groups
+      usersGroupSearchBase: cn=groups,ou=staff,cn=accounts,dc=sso,dc=example,dc=com
+      # backend.settings.auth.usersGroupSearchFilter -- Search filter to get users groups name
+      usersGroupSearchFilter: (&(uniqueMember={{dn}})(objectClass=posixGroup))
+      # backend.settings.auth.usersCustomersSearchBase -- Search base to find customers users
+      usersCustomersSearchBase: ou=customers,cn=accounts,dc=sso,dc=example,dc=com
+      # backend.settings.auth.usersCustomersSearchFilter -- Search filter to get customers users name
+      usersCustomersSearchFilter: (uid={{username}})
+      # backend.settings.auth.usersCustomersSearchBase -- Search base to find customers users group
+      usersCustomersGroupSearchBase: cn=groups,ou=customers,cn=accounts,dc=sso,dc=example,dc=com
+      # backend.settings.auth.usersCustomersGroupSearchFilter -- Search filter to get customers users groups name
+      usersCustomersGroupSearchFilter: (&(uniqueMember={{dn}})(objectClass=posixGroup))
+    ldap:
+      # backend.settings.ldap.host -- LDAP host
+      host: ldaps://sso.example.com:636
+      # backend.settings.ldap.bindDN -- LDAP bindDN to authenticate to LDAP server
+      bindDN: uid=customercenter-test.ldap.auth,cn=systems,dc=sso,dc=example,dc=com
+      # backend.settings.ldap.bindCredentials -- LDAP password to authenticate to LDAP server according to bindDN
+      bindCredentials: bind-credentials
+
+frontend:
+  # frontend.replicaCount -- (int) Number of Backend replicas
+  replicaCount: 1
+  image:
+    # frontend.image.repository -- Frontend image name
+    repository: docker.pkg.github.com/adfinis-sygroup/customer-center/frontend
```

Comment:
we should be moving this to ghcr.io but it's not scope of this PR.

Reply:
```suggestion
    repository: ghcr.io/adfinis-sygroup/customer-center-frontend
```

Reply:
I know it's not yet done (https://github.com/adfinis-sygroup/customer-center/pull/199) but as soon as it's merged, we should be able to switch to ghcr.io

Reply:
I will prepare it in advance right now.